src/CommonItilObject_Item.php

```diff
@@ -1576,4 +1577,36 @@ public static function dropdownAllDevices(
         }
         return $rand;
     }
+
+    /**
+     * ITIL tabs for generic assets should only be displayed if the asset already
+     * has associated ITIL items OR if the current user profile is allowed to
+     * link this asset to ITIL items
+     *
+     * @param Asset $asset
+     *
+     * @return bool
+     */
+    protected function shouldDisplayTabForAsset(Asset $asset): bool
+    {
+        // Always display tab if the current profile is allowed to link ITIL
+        // items to this asset
+        if (
+            in_array(
+                $asset::class,
+                $_SESSION["glpiactiveprofile"]["helpdesk_item_type"] ?? []
+            )
+        ) {
+            return true;
+        }
```

Comment:
It may be a problem. People may not be allowed to link computers to ITIL objects, but may want to be able to see existing links.

Reply:
This is covered by the check right after.

Reply:
Indeed.